lib/rajska.ex

```diff
@@ -23,19 +23,26 @@ defmodule Rajska do
 
   ## Usage
 
-  Create your Authorization module, which will implement the `Rajska.Authorization` behaviour and contain the logic to validate user permissions and will be called by Rajska middlewares. Rajska provides some helper functions by default, such as `c:Rajska.Authorization.is_role_authorized?/2`, `c:Rajska.Authorization.has_user_access?/4` and `c:Rajska.Authorization.is_field_authorized?/3`, but you can override them with your application needs.
+  Create your Authorization module, which will implement the `Rajska.Authorization` behaviour and contain the logic to validate user permissions and will be called by Rajska middlewares. Rajska provides some helper functions by default, such as `c:Rajska.Authorization.role_authorized?/2`, `c:Rajska.Authorization.has_user_access?/4` and `c:Rajska.Authorization.field_authorized?/3`, but you can override them with your application needs.
 
   ```elixir
   defmodule Authorization do
     use Rajska,
-      roles: [:user, :admin],
+      valid_roles: [:all, :user, :admin],
+      all_role: :all,
+      super_roles: [:admin],
       default_rule: :default
   end
   ```
 
-  Note: if you pass a non Keyword list to `roles`, as above, Rajska will assume your roles are in ascending order and the last one is the super role. You can override this behavior by defining your own `c:Rajska.Authorization.is_super_role?/1` function or passing a Keyword list in the format `[user: 0, admin: 1]`.
+  Default values:
 
-  Note: if the `:default_rule` is not given, `:default` is used as it's value.
+  ```elixir
+  valid_roles: [],
+  all_role: :all,
```

Comment:
Does it make sense to have this `role` as default even though it's not defined in `valid_roles`?

Reply:
It doesn't, I will remove it